src/onefetch/info.rs

```diff
@@ -312,15 +312,21 @@ impl Info {
 
         let pad = title.len() + 2;
 
-        for (i, (author_name, author_nbr_commits, autor_contribution)) in
+        for (i, (author_name, author_email_opt, author_nbr_commits, autor_contribution)) in
             self.authors.iter().enumerate()
         {
-            if i == 0 {
+			let author = if let Some(author_email) = author_email_opt {
+				format!("{} <{}>", author_name, author_email)
+			}else {
+				author_name.to_owned()
+			};
+			
+			if i == 0 {
```

Comment:
```suggestion
            let author = if let Some(author_email) = author_email_opt {
                format!("{} <{}>", author_name, author_email)
            } else {
                author_name.to_owned()
            };

            if i == 0 {
```
I think this would fail the format check, otherwise :slightly_smiling_face: 

This project uses 4 spaces for indentation (please check the `.editorconfig` file).
You may want to install an EditorConfig plugin for your preferred text editor so that
your style conforms with this project's style, or any other project that has an `.editorconfig`
file.